webapp/templates/team/partials/submission.html.twig

```diff
@@ -99,10 +99,18 @@
                                         <th>Description</th>
                                         <td>{{ run.0.description(true) }}</td>
                                     </tr>
-                                    <tr>
-                                        <th>Runtime</th>
-                                        <td>{{ judgingRun.runtime }} sec</td>
-                                    </tr>
+                                    {% if showLimits %}
```

Comment:
I prefer to not have this option, and first come up with a good reason why this shouldn´t be public.

If there is a good reason we should also add tests for this to make sure that we don´t expose that information, for now I miss the change to also expose this in the API to keep those in line.